README.md

```diff
@@ -128,6 +128,7 @@ Each [`Authorino`](https://github.com/Kuadrant/authorino-operator/tree/main/conf
 | listener                 | [Listener](#listener)       | Specification of the authorization service (gRPC interface). | Required |
 | oidcServer               | [OIDCServer](#oidcserver)   | Specification of the OIDC service. | Required |
 | metrics                  | [Metrics](#metrics)         | Configuration of the metrics server (port, level). | Optional |
+| healthz                  | [Healthz](#healthz)         | Configuration of the health/readiness probe (port). | Optional |
```

Comment:
It's related to this issue: https://github.com/Kuadrant/authorino/issues/355

Reply:
Sorry, I still do not get it after reading that. I will try to be more clear. Why expose the port to the user? what is the value added? The operator can enable healthcheck in authorino and use the default port for it. Maybe the healthcheck port is wanted to be monitored externally?

Reply:
It is for the principle more than anything else. This is a configuration option in the operand. As we don't want users touching the deployment directly, we expose it in the API – in this case, with some type-checking added.

The same is done for other configuration options that arguably might be otherwise more rigid, e.g. the metrics endpoint port number (`spec.metrics.port`).

If this should be configurable in the operand in the first place, maybe it's a question we should ask in https://github.com/kuadrant/authorino.